addons/account/wizard/accrued_orders.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api, _, Command
+from odoo.tools import format_date, float_is_zero
+from odoo.exceptions import UserError
+from dateutil.relativedelta import relativedelta
+import json
+
+
+class AccruedExpenseRevenue(models.TransientModel):
+    _name = 'account.accrued.orders.wizard'
+    _description = 'Accrued Orders Wizard'
+
+    company_id = fields.Many2one('res.company', default=lambda self: self.env.company)
```

Comment:
How do you manage multi-company cases ?
What if I open your wizard with sale/purchase orders belonging to multiple companies ?